mass_mailing_unsubscribe_af/i18n/sv_SE.po

```diff
@@ -179,57 +179,59 @@ msgstr "Senast uppdaterad"
 #. module: mass_mailing_unsubscribe_af
 #: model:ir.model,name:mass_mailing_unsubscribe_af.model_mail_unsubscription
 msgid "Mail unsubscription"
-msgstr ""
+msgstr "E-post avanmälan"
 
 #. module: mass_mailing_unsubscribe_af
 #: model_terms:ir.ui.view,arch_db:mass_mailing_unsubscribe_af.af_nyhetsbrev_unsubscribe
 msgid "Mailing Subscriptions"
-msgstr ""
+msgstr "Prenumeration av E-post utskick"
 
 #. module: mass_mailing_unsubscribe_af
 #: model:ir.model.fields,field_description:mass_mailing_unsubscribe_af.field_mail_unsubscription__mass_mailing_id
 msgid "Mass mailing"
-msgstr ""
+msgstr "Massutskick"
 
 #. module: mass_mailing_unsubscribe_af
 #: model:ir.model.fields,help:mass_mailing_unsubscribe_af.field_mail_unsubscription__mass_mailing_id
 msgid "Mass mailing from which he was unsubscribed."
-msgstr ""
+msgstr "Massutskick som han avregistrerade sig från."
```

Comment:
Do we now its he/han here?
This might be a case where we should use hen?

Reply:
it says that "he was unsubscribed", depending on the word "he", I translated it, but we can change it to hen.